frontend/src/components/common/CheckboxItem/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { ChangeEvent } from 'react';
+
+import Checkbox, { CheckboxStyleProps } from '../Checkbox';
+
+import * as S from './styles';
+
+interface CheckboxItemProps extends CheckboxStyleProps {
+  id: string;
+  label: string;
+  isChecked: boolean;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  name?: string;
+  isDisabled?: boolean;
+}
+const CheckboxItem = ({ id, name, label, isChecked, onChange, isDisabled = false, $style }: CheckboxItemProps) => {
```

Comment:
```suggestion
const CheckboxItem = ({ label, ...rest }: CheckboxItemProps) => {

...

<Checkbox {...rest}/>
```
이렇게  Checkbox에게 넘겨주기 위해 받는 props들은  구조분해할당 이용해서 넘겨주면 제 경험상 Checkbox props가 변경되어도 props넘겨주는 부분은 변경하지 않아도 돼서 좋았어요

만약 나중에 리팩토링을 한다면 이 방법 추천해요

isDisabled의 기본값은 이걸 사용하는 Checkbox에서 설정하면 좋고요

Reply:
오 js로 rest를 접했을 때는 사용한 속성들을 너무 축약하는 느낌이라 잘 안 쓰게 됐는데 이제 어차피 ts를 쓰니까 인터페이스에서 어떤 속성을 썼는지 찾아볼 수 있겠네요! 꿀팁 감사합니다